README.md

```diff
@@ -0,0 +1,42 @@
+# #|: Pipe data to and from IRC
+
+Hashpipe lets you connect stdin and stdout to IRC with Unix pipes. For example, `sleep 5; echo "done" | hashpipe -s irc.freenode.net` messages `#hashpipe` when a long-running command finishes.
```

Comment:
nit: "messages the `#hashpipe` irc channel on the freenode network". More explicit early on